filters/ThirdParty/filter_222_ABPPersian/template.txt

```diff
@@ -6,16 +6,27 @@
 ! Attention: check if these rules are removed after update this filter
 ! https://github.com/AdguardTeam/AdguardFilters/issues/53365
 ! ||digikala.com^$third-party
-! https://github.com/AdguardTeam/AdguardFilters/issues/60229
-! ##*[class$="_ads"]:not(body):not(input)
-! ~flickr.com##*[class$="-ads"]:not(body):not(input)
 !
-! *web&utm_campaign
+! ##*[class$="_ads"]:not(body):not(input)
+! ##*[class$="_ads"]:not(body):not(input)
```

Comment:
It seems that these rules are the same, but as it's just a comment then I guess it doesn't matter.

Reply:
Yes. The main change - disabled dangerous rules.

Reply:
removed duplicate comment